### ISSUE TITLE
bug: social follow rel=me seems to be broken

### ISSUE THREAD

**Issue by LPardue** [theNewDynamic/gohugo-theme-ananke#772]
## Expected Behavior

The documentation at https://github.com/theNewDynamic/gohugo-theme-ananke/wiki/Social-media-network-setup#rel-attributes-on-follow-links suggests I can add a `rel = "me"` configuration parameter. Here is my config

```
  [params.ananke.social.mastodon]
  username = "SimmerVigor"
  profilelink  = "https://mastodon.social/@SimmerVigor"
  rel = "me"
``` 

and I'd expect the Mastodon link on https://lucaspardue.com/ to include the `me` relation

## Current Behavior

Here's a snippet of generated link on  https://lucaspardue.com/, it contains only `noopener`

```
<a href="https://mastodon.social/@SimmerVigor" target="_blank" rel="noopener" class="ananke-social-link link-transition mastodon link dib z-999 pt3 pt0-l mr1" title="follow on Mastodon - Opens in a new window" aria-label="follow on Mastodon - Opens in a new window">
```

## Possible Solution

No idea :)

## Steps to Reproduce

1. Visit https://lucaspardue.com
2. Check the Mastodon link
3.
4.

## The bigger picture (we need context)

hugo v0.135.0-f30603c47f5205e30ef83c70419f57d7eb7175ab+extended linux/amd64 BuildDate=2024-09-27T13:17:08Z VendorInfo=gohugoio
ananke git  submodule @ c05ef6f3c76299d05f69a67f7ef1df7daf40b43c

**Comment by davidsneighbour** [comment]
Thanks, that is a great error report. I'll check this later today and patch it.

**Comment by davidsneighbour** [comment]
This is fixed in v2.11.2 - https://github.com/theNewDynamic/gohugo-theme-ananke/releases/tag/v2.11.2

**Comment by LPardue** [comment]
Wow, quick work. Tested on the live website and confirmed it works. Thank you!
